app/models/channel.rb

```diff
@@ -351,6 +351,10 @@ def update_site_banner_lookup!(skip_site_banner_info_lookup: false)
     site_banner_lookup.sync!
   end
 
+  def has_valid_uphold_connection?
```

Comment:
What do you think of renaming this to `has_valid_uphold_connection_for_cards?`
We'll also want to make sure that the ID of the card is the same ID of the current `selected_wallet_provider` on the `publisher` If it doesn't, we should launch a `CreateUpholdChannelCardJob` job and have that create cards with the current `UpholdConnection`